db/migrate/20150714140850_remove_new_from_compute_attributes.rb

```diff
@@ -0,0 +1,21 @@
+class FakeComputeAttribute < ActiveRecord::Base
+  set_table_name 'compute_attributes'
+  serialize :vm_attrs, Hash
+end
+
+class RemoveNewFromComputeAttributes < ActiveRecord::Migration
+  def up
+    FakeComputeAttribute.all.each do |comp_attr|
+      attrs_name = ['nics', 'interfaces'].find do |name|
+        comp_attr.vm_attrs["#{name}_attributes"].present?
+      end
+      comp_attr.vm_attrs["#{attrs_name}_attributes"].delete("new_#{attrs_name}") if attrs_name
+      comp_attr.vm_attrs['volumes_attributes'].delete('new_volumes')
```

Comment:
```
StandardError: An error has occurred, this and all later migrations canceled:

undefined method `delete' for nil:NilClass/home/ares/Projekty/Zdrojaky/foreman/db/migrate/20150714140850_remove_new_from_compute_attributes.rb:13:in `block in up'
/home/ares/Projekty/Zdrojaky/foreman/db/migrate/20150714140850_remove_new_from_compute_attributes.rb:8:in `each'
/home/ares/Projekty/Zdrojaky/foreman/db/migrate/20150714140850_remove_new_from_compute_attributes.rb:8:in `up'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

I think there should be some logic to skip deletion if there are no such attributes (same for line 12).